modules/oc-adm-by-example-content.adoc

```diff
@@ -82,6 +82,14 @@ Deny a certificate signing request
 == oc adm copy-to-node
 Copies specified files to the node.
 
+.Example usage
```

Comment:
@zhouying7780 @ardaguclu This autogeneration is pulling back in some of the examples we manually removed in past releases because they weren't working. It looks like some of these are different than before. Can you PTAL at each of these and let me know if they are okay now, or if any are still incorrect and need to be removed?